tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -71,6 +71,52 @@ def test_fsx_lustre(
     _test_data_repository_task(remote_command_executor, mount_dir, bucket_name, fsx_fs_id, region)
 
 
+@pytest.mark.parametrize("deployment_type, per_unit_storage_throughput", [("PERSISTENT_1", 200)])
```

Comment:
Can you hardcode `deployment_type` and `per_unit_storage_throughput` in the config files instead of parameterizing them, since they aren't needed by the validation logic?